terraform/ec2/amis.tf

```diff
@@ -22,7 +22,7 @@ variable "ami_family" {
       otconfig_destination     = "/tmp/ot-default.yml"
       download_command_pattern = "wget %s"
       install_command          = "while sudo fuser /var/cache/apt/archives/lock /var/lib/apt/lists/lock /var/lib/dpkg/lock /var/lib/dpkg/lock-frontend; do echo 'Waiting for dpkg lock...' && sleep 1; done; echo 'No dpkg lock and install collector.' && sudo dpkg -i aws-otel-collector.deb"
-      start_command            = "sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -c \"$(echo -n 'CONFIGURATION_URI_PLACEHOLDER' | base64 -d)\" -a start"
+      start_command            = "sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -c \"$(echo -n 'CONFIGURATION_URI_PLACEHOLDER' | base64 -d)\" -f -adot.exporter.datadogexporter.deprecation -a start"
```

Comment:
There are more than one feature gate that need to be tested.  This needs to be configurable on a per-test basis so that we can have a test that validates the functionality of the gate.

Reply:
Agreed